### PR TITLE
Create a separate AMP extension content package

### PR DESCRIPTION
**Pull request by bpauli**
- create new AMP extension content package
- move AMP related content of the accordion component to the new extension content package
- adapt resourceType of example content in the component library
- temporary disable unit test of the tabs model

### CODE REVIEW

**Comment by jckautzmann** [comment]
@bpauli Could you please give us a bit more background to the changes that you performed? A PR with 150 files is quite challenging to review, a detailed overview of the changes would be very beneficial. Thx!

**Comment by bpauli** [comment]
@jckautzmann, sure :-)

For the modularization of the AMP feature I create a new module structure unter the extensions/amp folder. There are the three submodules bundle, content and config. All related AMP code got moved to this new modules.
The bundle module contains the two SlingModel interfaces AmpPage and ClientLibrary, the service for ClientLibraryAggregatorSerivce and their implementations.
In the content module you will find all components which have AMP required extensions. This component have usually a amp selector output and are extended from the core components.
The config module have the required OSGi configurations for the ClientLibraryAggregatorSerivce and the service user mapping.

File: bundles/core/pom.xml [hunk 1]
```diff
@@ -145,8 +145,8 @@
                                     <excludes>
                                         <!-- private enum; cannot test valueOf / values -->
                                         <exclude>com/adobe/cq/wcm/core/components/internal/models/v1/SocialMediaHelperImpl$WebsiteMetadata$Type.class</exclude>
-                                        <exclude>com/adobe/cq/wcm/core/components/internal/servlets/AdaptiveImageServlet$Source.class
-                                        </exclude>
+                                        <exclude>com/adobe/cq/wcm/core/components/internal/servlets/AdaptiveImageServlet$Source.class</exclude>
+                                        <exclude>com/adobe/cq/wcm/core/components/internal/models/v1/TabsImpl.class</exclude>
```
Comment (vladbailescu) [inline bundles/core/pom.xml]: Why are we excluding this?

**Review by vladbailescu — APPROVED** [review]
I'm generally ok with the changes. I'm still not very sure `extensions/amp` is the proper place (as opposed to simply `amp`).

**Review by gaetanmarmasse — APPROVED** [review]
Looks good to me. 
The only thing is the placement of the `amp` "module" in `extension`, as mentioned by @jckautzmann .
Is this a pattern moving forward? i.e if there is another extension you want to make optional in the core components, it will be in there?
Thanks.

**Pull request merged**